dotnet/src/Connectors/Connectors.Onnx/Connectors.Onnx.csproj

```diff
@@ -4,7 +4,7 @@
     <!-- THIS PROPERTY GROUP MUST COME FIRST -->
     <AssemblyName>Microsoft.SemanticKernel.Connectors.Onnx</AssemblyName>
     <RootNamespace>$(AssemblyName)</RootNamespace>
-    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;netstandard2.1</TargetFrameworks>
```

Comment:
Suggest keeping the 2.0 or adding support for both 2.0 and 2.1.

```suggestion
    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
```

Reply:
`Microsoft.ML.OnnxRuntimeGenAI` only support 2.1

Reply:
Just opened a PR bringing standard 2.0 to Onnx, hope they accept it.

Link: https://github.com/microsoft/onnxruntime-genai/pull/582

Reply:
Unresolving the conversation because OnnxRuntimeGenAI was recently updated to support netstandard2.0.

Reply:
There's no updated nupkg yet.

Reply:
0.3.0 released with `netstandard2.0` support